docs/manual/library/index.md

```diff
@@ -102,7 +102,11 @@ To add an AND criteria, such as **Creator** containing *jimi* AND BPM in the ran
 
 By default, a smart block will not contain repeated items, which will limit the duration of the block if you do not have sufficient items meeting the specified criteria in your **Library**. To override the default behaviour, check the **Allow Repeat Tracks** box. The **Sort tracks by** menu offers the options of **random**, **newest** or **oldest** items first.
 
-<img src="static/Screenshot582-Smart_block_repeat_tracks.png" alt="Smart block repeat tracks" width="595" height="255" />
+![](static/Screenshot582-Smart_block_repeat_tracks.png)
```

Comment:
You can but the alt text between the brackets:
```suggestion
![Smart block repeat tracks](static/Screenshot582-Smart_block_repeat_tracks.png)
```
I think that we used an `<img>` here is an artefact of the conversion from booktype.

Reply:
Yeah, this should have been converted as well during the great docs refactor. Re-adding the alt attribute should/could help with screen readers and i11y but isn't all that important.